.eslintrc.js

```diff
@@ -5,8 +5,9 @@ module.exports = {
   },
   extends: [
     'plugin:vue/essential',
+    // 'plugin:vue/recommended',
     '@vue/standard',
-    '@vue/typescript'
+    '@vue/typescript/recommended'
```

Comment:
I'm thinking lines 8 and 10 are worthy changes but I'm trying to limit the impact of the upgrade -- we can use Vue Recommended rules, and fix what it finds, later.